feat(atomic): migrating content from docs site to storybook (#6133)
coveo/ui-kit: packages/atomic/src/components/search/atomic-breadbox/atomic-breadbox.mdx

import { Meta } from '@storybook/addon-docs/blocks';
import * as AtomicBreadboxStories from './atomic-breadbox.new.stories';
import { AtomicDocTemplate } from '../../../../storybook-utils/documentation/atomic-doc-template';

<Meta of={AtomicBreadboxStories} />

<AtomicDocTemplate
  stories={AtomicBreadboxStories}
  githubPath="search/atomic-breadbox/atomic-breadbox.ts"
  tagName="atomic-breadbox"
  className="AtomicBreadbox"
>

Include breadcrumbs in a search page when you want users to navigate the content using facets.

> For eaxmple, on an ecommerce website breadcrumbs help the users keep track of the selected filters and being able to confirm and adjust filtering encourages browsing.

Reviewer: I'd remove this or replace it with a non-commerce example (since this component is in the "search" use case, no commerce).

Also, I'd add:

For the sake of consistency with other mdx, I'd add:

This component is typically placed within the "status" section of the layout.

```html
<atomic-search-interface>
  ...
  <atomic-search-layout>
    ...
    <atomic-layout-section section="main">
      ...
      <atomic-layout-section section="status">

        <atomic-breadbox></atomic-breadbox>
        
      </atomic-layout-section>
      ...
    </atomic-layout-section>
  </atomic-search-layout>
</atomic-search-interface>
```

for the sake of consistency with other mdx.

> 

Reviewer: ```suggestion
```

## UX Best Practices

> - **Field names provide context for the selected value**: displaying the field name next to the selected value clarifis which facet the breadcrumb value associated with.

Reviewer: ```suggestion
- **Field names provide context for the selected value**: displaying the field name next to the selected value clarifies which facet the breadcrumb value associated with.
```

> In the case where there are two or less facets, remove the field name through the CSS shadow part `breadcrumb-label` if the context is clear.

Reviewer: ```suggestion
If there are only one or two facets in the page with clearly distinct values (e.g., author names and file size ranges), you can remove the field name through the `breadcrumb-label` CSS shadow part (e.g., by styling it to `display: none`).
```
